com.stansassets.scene-management/BuildConfigurator/Runtime/BuildConfiguration.cs

```diff
@@ -36,7 +42,7 @@ internal bool UseAddressablesInEditor
             get
             {
 #if UNITY_EDITOR
-                return UnityEditor.EditorPrefs.GetBool($"{Name}_user-addressables-in-editor", false);
```

Comment:
I think it would be a great idea to move those string literals to private const strings.
Thus, we won't get them duplicated throughout this file.